core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -136,6 +137,7 @@ public class RESTSessionCatalog extends BaseViewSessionCatalog
   private FileIO io = null;
   private MetricsReporter reporter = null;
   private boolean reportingViaRestEnabled;
+  private String restPageSize = null;
```

Comment:
We should make this `Integer` not `String`

Reply:
will fix